org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+  "ava": {
+    "files": [
+      "tests/*.js"
+    ]
+  },
+  "dependencies": {
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0"
+  },
+  "devDependencies": {
+    "delay": "^1.3.1",
+    "jshint": "^2.9.2",
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
```

Comment:
Sure. I had changed this, since Jean had similar feedback. I shall change it back to thermostat.js and hope to meet tomorrow to flush out final design.

---
In reply to: [73946648](https://github.com/openT2T/translators/pull/37#discussion_r73946648) [](ancestors = 73946648)